travispy/_helpers.py

```diff
@@ -38,3 +41,24 @@ def get_response_contents(response):
     else:
         contents['status_code'] = status_code
         raise TravisError(contents)
+
+
+def get_archived_log(session, job_id):
```

Comment:
I wasnt sure where this utility should go.  I'd like it to be callable without instantiating a Log, etc. so that tools can use it directly if they have a job_id.